### PR TITLE
Add some simple validation

### PR DESCRIPTION
**Pull request by dshiga**
Uses the wdl parser to check payload for syntax errors when posting. Not sure we should merge yet because we're going to be getting much more thorough validation capabilities soon from the cromwell repo.

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/agora/server/webservice/methods/MethodsService.scala [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package org.broadinstitute.dsde.agora.server.webservice.methods
 
+import java.util.Scanner
```
Comment (bradtaylor) [inline src/main/scala/org/broadinstitute/dsde/agora/server/webservice/methods/MethodsService.scala]: Are Scanner and WdlParser used in this class?

Comment (dshiga) [inline src/main/scala/org/broadinstitute/dsde/agora/server/webservice/methods/MethodsService.scala]: Ah, good catch. I'll remove those imports.

**Comment by bradtaylor** [comment]
This looks awesome.

File: .travis.yml [hunk 2]
```diff
@@ -3,5 +3,7 @@ scala:
   - 2.11.6
 services:
   - mongodb
+jdk:
+  - oraclejdk8
```
Comment (bradtaylor) [inline .travis.yml]: Does the WDL parser require java 8?  We are pulling java 7 into the docker image.

Comment (dshiga) [inline .travis.yml]: It does in its current form. We *could* make it work with 7 but that adds some complications.

**Pull request merged**